INTG-1582 Use lorawan decoding package

mateuszpohl opened the pull request.

KevinJoiner reviewed (commented):
Are you going to update the signal conversion as well? If not I am confused on why we added this generated code.
https://github.com/DIMO-Network/model-garage/blob/main/pkg/lorawan/status/payload-status-v2.go#L16

mateuszpohl commented:
> Are you going to update the signal conversion as well? If not I am confused on why we added this generated code. https://github.com/DIMO-Network/model-garage/blob/main/pkg/lorawan/status/payload-status-v2.go#L16

Signal conversion is also updated, the status package switched from `autopi/status` to `lorawan/status` so the only change needed was the import.

zakharenkodmytro reviewed (approved):
lgtm
